packages/client/test/runner.js

```diff
@@ -2,18 +2,21 @@
 
 const { tap, spec } = require('node:test/reporters')
 const { run } = require('node:test')
-const path = require('node:path')
 const glob = require('glob').globSync
 
 /* eslint-disable new-cap */
 const reporter = process.stdout.isTTY ? new spec() : tap
 
-const files = [
-  ...glob(path.join(__dirname, '*.test.js'))
-]
+const files = glob('test/*.test.{js,mjs}')
```

Comment:
I don’t like this very much, as the runner could be executed only from the root of the package.